### ISSUE TITLE
SAR versioning is behind

### ISSUE THREAD

**Issue by jeremiahlukus** [alexcasalboni/aws-lambda-power-tuning#250]
```
{
  "name": "Branching",
  "output": {
    "lambdaARN": "arn:aws:lambda:us-east-1:blah:function:staging-controller",
    "powerValues": [
      128,
      256,
      512,
      1024,
      2048,
      3008
    ],
    "num": 5,
    "payload": {},
    "parallelInvocation": false,
    "strategy": "cost",
    "error": {
      "Error": "SyntaxError",
      "Cause": "{\"errorType\":\"SyntaxError\",\"errorMessage\":\"Unexpected token 'W', \\\"WARN: hand\\\"... is not valid JSON\",\"trace\":[\"SyntaxError: Unexpected token 'W', \\\"WARN: hand\\\"... is not valid JSON\",\"    at JSON.parse (<anonymous>)\",\"    at /var/task/utils.js:551:54\",\"    at Array.map (<anonymous>)\",\"    at module.exports.extractDurationFromJSON (/var/task/utils.js:551:35)\",\"    at module.exports.extractDuration (/var/task/utils.js:528:22)\",\"    at /var/task/utils.js:472:22\",\"    at Array.map (<anonymous>)\",\"    at module.exports.parseLogAndExtractDurations (/var/task/utils.js:470:17)\",\"    at computeStatistics (/var/task/executor.js:180:29)\",\"    at module.exports.handler (/var/task/executor.js:71:12)\"]}"
    }
  },
  "outputDetails": {
    "truncated": false
  }
}
```

Created using the aws GUI 

https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:451282441545:applications~aws-lambda-power-tuning

**Comment by jeremiahlukus** [comment]
last release was `4.3.4 (2024-02-26):` maybe doing a new release will fix the issue?

**Comment by jeremiahlukus** [comment]
After using sam to deploy i no longer get this error. I think just doing another release to AWS would fix.

**Comment by alexcasalboni** [comment]
Hi @jeremiahlukus thanks for flagging this!

We are aware that SAR versioning is a bit behind - we're waiting to get a major release out for #206 (something like `5.0.0`). Apologies for the trouble, but glad you managed to use SAM to deploy the latest in the meantime.

I'm keeping this issue open for now, even though the bug has been resolved already. Will share an update and close once the new SAR version is out.

**Comment by alexcasalboni** [comment]
@jeremiahlukus apologies for the long wait :)

I've just published a new `4.3.5` version to SAR 👌 

Closing this issue.
